pennylane_qiskit/qiskit_device.py

```diff
@@ -22,8 +22,9 @@
 import warnings
 
 import numpy as np
+import qiskit
 from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister
-from qiskit import extensions as ex
+from qiskit.circuit import library as lib
```

Comment:
to be sure, this existed in qiskit v0.x as well? do you know the minimum version at which it was available as `qiskit.circuit.library`?

Reply:
Yep. Has been there since [v0.24](https://docs.quantum.ibm.com/api/qiskit/0.24/circuit_library) at least.